src/liquid/components/ld-switch/readme.md

```diff
@@ -0,0 +1,567 @@
+---
+eleventyNavigation:
+  key: Switch
+  parent: Components
+layout: layout.njk
+title: Switch
+permalink: components/ld-switch/
+---
+
+<link rel="stylesheet" href="css_components/ld-switch.css">
+<link rel="stylesheet" href="css_components/ld-switch-item.css">
+<link rel="stylesheet" href="css_components/ld-icon.css">
+
+# ld-switch
+
+A switch is a collection of two or more switch items. It can have a single or no item selected at a time.
```

Comment:
```suggestion
A switch is a collection of two or more items that behaves similar to a radio button group or toggle. It can have a single or no item selected at a time.
```